docs/metadata.rst

```diff
@@ -0,0 +1,98 @@
+Import metadata using the Command Line Interface (CLI)
+=======================================================
+
+Description:
+------------
+
+This chapter will show how to import metadata starting from a local CSV file and ending with OMERO.tables on images or Key-Value pairs on images using the Command Line Interface (CLI) and the OMERO server-side scripts.
+
+This action is typically done after a successful import of images.
+
+We will show:
+
+-  How to import metadata from local CSV file in a bulk manner and turn them into OMERO.tables on images using CLI
+
+- How to turn the OMERO.tables on images into Key-Value pair annotations on images in bulk manner using CLI
+
+- How to import metadata from local CSV file and use a server-side script in OMERO to turn these into OMERO.tables on images
+
+- How to construct a simple file to turn the metadata stored in OMERO.tables into Key-Value pairs on images using CLI
+
+**Resources:**
+--------------
+
+-  Documentation:
+
+   -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html
```

Comment:
Perhaps use the same omero/latest or latest/omero everywhere. (see next line)

Reply:
Fixed in https://github.com/ome/omero-guide-upload/pull/6/commits/321c12605df82d3ab121873b8eb4ca546a87ce74